### DISCUSSION TITLE
Questions about the Aqara C1 Pet Feeder (ZNCWWSQ01LM) Home Assistant integration

### DISCUSSION THREAD

**Discussion by brubaked** [Koenkk/zigbee2mqtt#15613] [Devices]
All the controls (that I have tested - haven't gotten around to testing schedule yet) for the C1 pet feeder work great and as expected from the Zigbee2MQTT add-on web ui. I can hit "start" from the exposes page n-times in a row and it will dispense n-times the serving size. 

On the Home Assistant side, using the "select.friendly_name_feed" entity will only dispense a serving if the state of the entity is not already "Start". For example, if the state is "Unknown", or I manually change it to something like "." using the developer tools, I can change "select.friendly_name_feed" to "Start" and it will dispense a serving. After selecting "Start" once, reselecting "Start" will not initiate another feeding. 

Is there a way to approach controlling the feeder from Home Assistant that allows reliable initation of feeding events?

**Comment by RaulVKoNe** [comment]
Same problem
